soda-consumer-scala/src/main/scala/com/socrata/soda2/consumer/http/HttpConsumer.scala

```diff
@@ -10,8 +10,6 @@ import com.socrata.future.ExecutionContextTimer
 
 /** An implementation of [[com.socrata.soda2.consumer.Consumer]] which operates on a real HTTP server. */
 class HttpConsumer(lowLevel: LowLevelHttp) extends Consumer(lowLevel) {
-  private[socrata] def this(client: AsyncHttpClient, logicalHost: String, physicalHost: String, port: Int = 443, secure: Boolean = true, authorization: Authorization = NoAuth)(implicit executionContext: ExecutionContext, timer: ExecutionContextTimer) =
```

Comment:
Both this and the private constructor for HttpPublisher were error-ing with " multiple overloaded alternatives of constructor HttpConsumer define default arguments." This [isn't allowed](https://stackoverflow.com/questions/4652095/why-does-the-scala-compiler-disallow-overloaded-methods-with-default-arguments).